next-app/src/lib/stats/index.ts

```diff
@@ -1,6 +1,7 @@
 import Stats from './Stats.svelte'
 import Stat from './Stat.svelte'
 
+// @ts-expect-error, not sure how to correct this one via typescript
```

Comment:
I had done it that way to provide a _hint_ to the consumer that `Stat` should only be used inside a `Stats`.  If we move them out like you're suggesting, I suppose I'm concerned it will force more education/convention...

let's talk more about this

Reply:
corrected here:  [cc966de](https://github.com/sillsdev/web-languageforge/pull/1664/commits/cc966de265900e2f98a35d4ae29d93873150116e)